gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -2065,6 +2084,17 @@ public void close() {
     }
   }
 
+  private void closeCloudLogging() {
+    if (cloudLogger != null) {
+      try {
+        cloudLogger.flush();
```

Comment:
This is synchronous operation that can increase shutdown time. Also, does it have timeout?